### ISSUE TITLE
Error "Collection File ... is missing from model" when using  repeatable component

### ISSUE THREAD

**Issue by aquaswim** [arrowheadapps/strapi-connector-firestore#57]
## Error:

When using component that have media field always throw error that File model is not found
```
[2021-07-12T06:43:09.283Z] error Something went wrong in the model `Application::benefit.benefit` with the attribute `benefitItems$meta.icon`
[2021-07-12T06:43:09.285Z] error Error: The collection `File`, used in the attribute `benefitItems$meta.icon` in the model Application::benefit.benefit, is missing from the models
    at Object.getNature (/Users/skyshi/code/node/iris-cms/node_modules/strapi-utils/lib/models.js:79:15)
    at Object.defineAssociations (/Users/skyshi/code/node/iris-cms/node_modules/strapi-utils/lib/models.js:373:26)
    at mountModel (/Users/skyshi/code/node/iris-cms/node_modules/strapi-connector-firestore/lib/model.js:155:22)
    at Object.mountModels (/Users/skyshi/code/node/iris-cms/node_modules/strapi-connector-firestore/lib/model.js:54:21)
    at /Users/skyshi/code/node/iris-cms/node_modules/strapi-connector-firestore/lib/index.js:88:21
    at async Promise.all (index 0)
    at async Object.initialize (/Users/skyshi/code/node/iris-cms/node_modules/strapi-connector-firestore/lib/index.js:55:9)
    at async Object.initialize (/Users/skyshi/code/node/iris-cms/node_modules/strapi-database/lib/connector-registry.js:30:9)
    at async DatabaseManager.initialize (/Users/skyshi/code/node/iris-cms/node_modules/strapi-database/lib/database-manager.js:43:5)
    at async Strapi.load (/Users/skyshi/code/node/iris-cms/node_modules/strapi/lib/Strapi.js:354:5)

```

## dependencies in package.json

```
    {
    "lodash": "^4.17.21",
    "strapi": "3.6.5",
    "strapi-admin": "3.6.5",
    "strapi-connector-firestore": "^3.0.0-alpha.35",
    "strapi-plugin-content-manager": "3.6.5",
    "strapi-plugin-content-type-builder": "3.6.5",
    "strapi-plugin-email": "3.6.5",
    "strapi-plugin-upload": "3.6.5",
    "strapi-plugin-users-permissions": "3.6.5",
    "strapi-provider-upload-google-cloud-storage": "^3.6.3",
    "strapi-utils": "3.6.5"
  }
```

## How to reproduce

Create content type that have component inside it, and the component have media field.

**Comment by brettwillis** [comment]
@aquaswim thank you for the report, I appreciate it! I'll try to take a look within a few days to a week.

**Comment by brettwillis** [comment]
@aquaswim I've fixed this issue in `3.0.0-alpha.36`, please give it a try, and let me know how it goes. Thanks!

**Comment by aquaswim** [comment]
yeah, everything is working now, thanks 👍
